xml/cap_depl_azure.xml

```diff
@@ -412,10 +398,12 @@ done</screen>
   </para>
 
 <screen>&prompt.user;helm repo list
-NAME            URL
-stable          https://kubernetes-charts.storage.googleapis.com
-local           http://127.0.0.1:8879/charts
-suse            https://kubernetes-charts.suse.com/</screen>
+NAME       URL
+stable     https://kubernetes-charts.storage.googleapis.com    
+local      http://127.0.0.1:8879/charts                        
+suse       https://kubernetes-charts.suse.com/                 
+svc-cat    https://svc-catalog-charts.storage.googleapis.com   
+azure      https://kubernetescharts.blob.core.windows.net/azure</screen>
 
```

Comment:
This one is a minor point, but if we're assuming the user is only preparing to add the `suse` repo to deploy on AKS then they normally wouldn't have the `svc-cat` and `azure` repos as they haven't reached the AKS service broker section (that adds these repos) yet.

Reply:
Good point. I reran all the helm commands to test them, but I did not start with an empty configuration.